src/base_alloc/base_alloc_global.h

```diff
@@ -17,6 +19,7 @@ extern "C" {
 void *umf_ba_global_alloc(size_t size);
 void umf_ba_global_free(void *ptr);
 void umf_ba_destroy_global(void);
+bool umf_ba_global_is_destroyed();
```

Comment:
Where it is used?

Reply:
right now nowhere but it would be needed when Disjoint Pool would use base alloc (in C version) in umfDisjointPoolParamsDestroy

Reply:
So should we close this PR?